atm.js

```diff
@@ -1,21 +1,118 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance = $(".balance")[0];
+var checkingInt = Number.parseInt(checkingBalance.innerHTML.replace("$", ""));
+$(".balance").eq(0).addClass("zero");
+// console.log(checkingInt);
+var savingsBalance = $(".balance")[1];
+var savingsInt = Number.parseInt(savingsBalance.innerHTML.replace("$", ""));
+$(".balance").eq(1).addClass("zero");
+// console.log(savingsInt);
+
+
+// add eventListener to each button
+$("[value=\"Deposit\"]").eq(0).click( function(event){checkingDeposit(event);} );
+$("[value=\"Withdraw\"]").eq(0).click( function(event){checkingWithdrawal(event);} );
+
+// event listeners for savings account buttons
+$("[value=\"Deposit\"]").eq(1).click( function(event){savingsDeposit(event);} );
+$("[value=\"Withdraw\"]").eq(1).click( function(event){savingsWithdrawal(event);} );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-}
 
-function checkingWithdrawal(amount){
+function checkingDeposit(event){
+  // Return if text input is empty string
+  if($("[type=\"text\"]").eq(0).val() === ""){
+    return;
+  }
+  // Check contents (integer form) of text input field
+  checkingInt = Number.parseInt(checkingBalance.innerHTML.replace("$", ""));
+  // Sum together that number and the number parsed from the current balance
+  checkingBalance.innerHTML = "$" + (checkingInt + Number.parseInt($("[type=\"text\"]").eq(0).val()));
+  // Clear the text input field
+  $("[type=\"text\"]").eq(0).val("");
 
+  // if the checking account balance > 0, add the class 'zero'
+  if(Number.parseInt(checkingBalance.innerHTML.replace("$", "")) > 0){
+    $(".balance").eq(0).removeClass("zero");
+  }
 }
```

Comment:
might be good to encapsulate in this method, because you will be repeating this functionality any time you change the balance. Additionally, i think its only neccessary for the withdrawals.